python/py-tensorflow_estimator/Portfile

```diff
@@ -34,7 +33,7 @@ livecheck.regex     /project/tensorflow-estimator/(\\d+(\\.\\d+)+)/
 extract.suffix      .whl
 extract.only
 
-python.versions     37 38 39 310
```

Comment:
Any reason not to leave 37 here as the page at https://pypi.org/project/tensorflow-estimator/ says `Requires: Python >=3.7`

Reply:
Python 37 is well beyond EOL, and that requirement is **greater-than**-or-equal-to Python 37; therefore, Python 37 is unnecessary.

But you're doing the work of the port maintainer, so if you'd like it to stay, then I'd say it's up to you.

Reply:
Additionally, MacPorts has no effective support for `py37-tensorflow` anymore because the upstream build chains for both tensorflow and bazel are broken and have been for some time. And there is no Python 37 support in `py-tensorflow-macos`.

Therefore, we should remove all Python 37 versions for the tensorflow chain.